.devcontainer/devcontainer.json

```diff
@@ -1,21 +1,34 @@
 {
+  // See the devcontainer spec at https://containers.dev/implementors/spec/
+  "name": "React Codespaces Template",
+
+  // Container image to build environment from
+  // universal contains many tools making it useful in many contexts https://github.com/devcontainers/images/tree/main/src/universal
   "image": "mcr.microsoft.com/devcontainers/universal:2",
+
+  // Specify the minimum machine spec for running the dev container
   "hostRequirements": {
     "cpus": 4
   },
-  "waitFor": "onCreateCommand",
+
+  // Upon Codespaces or image prebuild, refresh the npm and node.hs installation (async from environment creation)
   "updateContentCommand": "npm install",
-  "postCreateCommand": "",
```

Comment:
I wonder if this was explicitly added to avoid the oryx tool from running? 🤔

Reply:
Yup it was 👍

Reply:
That is a dirty hack! No other way to do that?